src/workspacer.Native/Native/WindowsDeferPosHandle.cs

```diff
@@ -67,7 +67,12 @@ public void DeferWindowPos(IWindow window, IWindowLocation location)
                 _toNormal.Add(window);
             }
 
-            Win32.DeferWindowPos(_info, window.Handle, IntPtr.Zero, location.X, location.Y, location.Width, location.Height, flags);
+            int X = location.X + window.Offset.X;
+            int Y = location.Y + window.Offset.Y;
+            int Width = location.Width + window.Offset.Width;
+            int Height = location.Height + window.Offset.Height;
```

Comment:
I guess this is the "meat" of the fix which removes the gaps. But reading the code, it's not entirely obvious *how*.

My intuitive understanding would be that to remove the gaps we need to make the values of `X` and `Y` smaller (and to do that we should *subtract* something), while we want to make `Width` and `Height` bigger (by *adding* something).

Is my understanding correct?

If so, here we are adding in both cases, and there is no subtraction to the `X` and `Y` variables. Is `Offset.X` and `Offset.Y` expected to return a negative values, while `Offset.Width` and `Offset.Height` return positive ones? 

This section could benefit from values/forms better matching the mental model of how one fills the gaps, or by adding more comments/documentation outlining what is supposed to happen here.

Reply:
Also: In this section you are pulling in `Offset` 4 times to access 4 sub-properties. This means you actually calculate the full Offset variable once for each and every one of those sub-properties.

Assign `Offset` to a local variable and access sub-properties from that one instance instead.

Reply:
This is clearly much better now. I still think it's a little unintuitive to *add* the X and Y offsets, but on the other hand, it wouldn't make much sense either to return a rectangle where we change up the X and Y without ... Also adjusting the height and width.

So yeah. Let's keep it as is then.